### ISSUE TITLE
alt text on mathquill palette buttons

### ISSUE THREAD

**Issue by Alex-Jordan** [openwebwork/mathquill#24]
Our disability services team reviewed some things recently, and one thing they report is that assistive technology voicing the palette tool buttons is off. For example, both the square root and nth root buttons were spoken as "square root".

**Comment by drgrice1** [comment]
I don't think this is a mathquill issue.  The "palette" that you are referring to is not part of mathquill.  This is probably a PG issue.  I might be mistaken though.

**Comment by Alex-Jordan** [comment]
I'm referring to this thing:

<img width="199" alt="Screenshot 2024-03-18 at 6 54 33 PM" src="https://github.com/openwebwork/mathquill/assets/4732672/67e8ef78-7ed6-470a-80f5-43512093199d">

Is that not part of MathQuill? Even though the tooltip for nth root looks correct, the report is that some screen reader (they didn't say what they tested with in the email) was saying "square root" on that one. So we might be missing some attribute to make it say the right thing and not try to read the actual `√` character that it is in there. They only mentioned the nth root as an example, but the email makes it sounds like there are more instances of button speech not matching the tooltip.

**Comment by drgrice1** [comment]
No, that is part of PG, not MathQuill.  That is constructed by the `htdocs/js/MathQuill/mqeditor.js` file.

Perhaps what is needed is to add `aria-hidden` to the static MathQuill that is in the button, and change the `data-bs-title` attribute to an actual `title` attribute.  That will work the same for the tooltip, and that is then what the screen reader should read.

**Comment by Alex-Jordan** [comment]
This was not a MQ issue, and has been fixed.
